wikimedia/wikipedia-ios
# Fastlane: PR creation for version bumps

**Phabricator:** N/A

### Notes
* This automates version changes, but requiring human approval.
* After this is approved, I need to change the branch that Jenkins points to. (https://wmf2249:8043/job/Increase%20version/configure). Additionally, I need to merge two Jenkins jobs - this one and `Tag last beta as release`, so they happen as one.

### Test Steps
1. Log into Jenkins.
2. Run the `Increase version` job. 
3. Ensure it opens a PR, and it looks appropriate. Close the PR, and delete the branch (via GH).
